src/sass/_Fabric.Color.Variables.scss

```diff
@@ -66,14 +67,13 @@ $ms-color-green:         #107c10;
 $ms-color-greenLight:    #bad80a;
 
 
-//== Other Colors
+//== Message colors
 //
-$ms-color-error:             $ms-color-redDark;
-$ms-color-errorBackground:   #fde7e9;
+$ms-color-info:              $ms-color-green;
+$ms-color-infoBackground:    $ms-color-neutralLighter;
 $ms-color-success:           $ms-color-green;
 $ms-color-successBackground: #dff6dd;
```

Comment:
Oh I see. Those actually are variables and are with the others in `_Fabric.Color.Variables.scss`. It's just that some of the other message color variables are mapped to existing colors.